manager/TODO.py

```diff
@@ -0,0 +1,23 @@
+
```

Comment:
I'd keep the TODO list out of the code, ideally this should be in GH issues, so that others can be assigned etc.